vmdb/app/models/pg_lock.rb

```diff
@@ -0,0 +1,29 @@
+class PgLock < ActiveRecord::Base
```

Comment:
I think the consistency might be more important than a one-line table_name (plus it puts all the files next to each other in file viewers).  I personally *actually* prefer your naming, but I think renaming everything else to look like PgStatActivity, or whatever, is a bigger change and should probably all be done at once, separately..